chart/epinio/templates/cert-secrets.yaml

```diff
@@ -0,0 +1,93 @@
+{{- if not .Values.certManager.enabled }}
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  annotations:
+  name: epinio-registry-tls
+  namespace: {{ .Release.Namespace }}
+stringData:
+  ca.crt: {{ .values.certManager.registry.ca | quote }}
+  tls.key: {{ .values.certManager.registry.key | quote }}
+  tls.crt: {{ .values.certManager.registry.cert | quote }}
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  annotations:
+  name: epinio-tls
+  namespace: {{ .Release.Namespace }}
+stringData:
+  ca.crt: {{ .values.certManager.epinio.ca | quote }}
+  tls.key: {{ .values.certManager.epinio.key | quote }}
+  tls.crt: {{ .values.certManager.epinio.cert | quote }}
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  annotations:
+  name: epinio-ca-root
+  namespace: {{ .Release.Namespace }}
+stringData:
+  ca.crt: {{ .values.certManager.caRoot.ca | quote }}
+  tls.key: {{ .values.certManager.caRoot.key | quote }}
+  tls.crt: {{ .values.certManager.caRoot.cert | quote }}
+{{- if .Values.global.dex.enabled }}
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  annotations:
+  name: dex-tls
+  namespace: {{ .Release.Namespace }}
+stringData:
+  ca.crt:  {{ .values.certManager.dex.ca   | default .values.certManager.epinio.ca   | quote }}
+  tls.key: {{ .values.certManager.dex.key  | default .values.certManager.epinio.key  | quote }}
+  tls.crt: {{ .values.certManager.dex.cert | default .values.certManager.epinio.cert | quote }}
+{{- end }}
+{{- if and .Values.epinioUI.enabled .Values.epinioUI.ingress.enabled }}
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  annotations:
+  name: epinio-ui-tls
```

Comment:
Shared `epinio` ingress for `epinio-server` and `epinio-ui` is using `epinio-tls` only.

Reply:
Yes, I think that we can completely drop the `epinio-ui-tls` certificate completely.
I would do this in a separate PR that should be merged before this one.
It will simplify the helm charts, and this PR.

Reply:
See PR #511.